ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -13,6 +13,151 @@ namespace ext
 namespace MitsubaLoader
 {
 
+static uint64_t rgb32f_to_rgb19e7(const uint32_t _rgb[3])
```

Comment:
put this in core, where the rgb9e5, rg11b10f and half-float compressors are

Reply:
still an issue

Reply:
will do once it's implemented correctly